tests/unit/amo/containers/TestHome.js

```diff
@@ -6,21 +6,24 @@ import {
 } from 'react-addons-test-utils';
 import { Provider } from 'react-redux';
 
+import { HomeBase } from 'amo/containers/Home';
 import createStore from 'amo/store';
+import { setClientApp, setLang } from 'core/actions';
 import { getFakeI18nInst } from 'tests/unit/helpers';
-import Home from 'amo/containers/Home';
 
 
 describe('Home', () => {
   it('renders a heading', () => {
-    const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
-    const { store } = createStore(initialState);
+    const { store } = createStore();
+    store.dispatch(setClientApp('android'));
```

Comment:
It works with `dispatchSignInActions` so I'll use that for now if it's okay. Technically it's dispatching more than is needed but that's okay.